assets/dev/js/editor/atomic-widgets/manager.js

```diff
@@ -1,6 +1,6 @@
-import ElementBase from './types/base/element-base';
+import ElementBase from '../elements/types/base/element-base';
```

Comment:
@baghdasarovelementor Why did you rename this file? If it was because of the comments on my other PR, what I meant there was that only AContainerClass(es) needs to be moved into the atomic folder. And you should use `registerElementType()` to register it, instead of adding it directly into the base. So everything that is specific to the atomic container should be in the atomic folder.

Reply:
Just to clarify, I want to see if all the JS can be in `/modules/atomic-widgets/assets/js/editor`

Reply:
@baghdasarovelementor ☝️ ?

Reply:
I made change you request in previous comment. Now I am on failing tests. @louiswol94